lib/style/single_node.ex

```diff
@@ -95,6 +95,29 @@ defmodule Styler.Style.SingleNode do
   defp style({{:., dm, [{:__aliases__, am, [:Logger]}, :warn]}, funm, args}),
     do: {{:., dm, [{:__aliases__, am, [:Logger]}, :warning]}, funm, args}
 
+  # Transform Timex defdelegates
+  defp style({{:., dm, [{:__aliases__, am, [:Timex]}, :today]}, funm, args}),
+    do: {{:., dm, [{:__aliases__, am, [:Date]}, :utc_today]}, funm, args}
+
+  defp style({{:., dm, [{:__aliases__, am, [:Timex]}, :now]}, funm, args}),
+    do: {{:., dm, [{:__aliases__, am, [:DateTime]}, :utc_now]}, funm, args}
```

Comment:
let me sit on these ones -- i'd considered doing the same earlier, but nixed it as too specific to adbe internals 🤔 still, this repo is all about forcing our views on others xD